frontend/src/component/project/ProjectCard/ProjectCardFooter/ProjectCardFooter.tsx

```diff
@@ -0,0 +1,58 @@
+import type { VFC } from 'react';
+import { Box, styled } from '@mui/material';
+import { FavoriteIconButton } from 'component/common/FavoriteIconButton/FavoriteIconButton';
+import useToast from 'hooks/useToast';
+import { useFavoriteProjectsApi } from 'hooks/api/actions/useFavoriteProjectsApi/useFavoriteProjectsApi';
+import useProjects from 'hooks/api/getters/useProjects/useProjects';
+
+interface IProjectCardFooterProps {
+    id: string;
+    isFavorite?: boolean;
+}
+
+const StyledFooter = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    padding: theme.spacing(1, 2),
+    borderTop: `1px solid ${theme.palette.grey[300]}`,
+    backgroundColor: theme.palette.grey[100],
```

Comment:
How does this work in dark mode? Feels kinda weird to use the palette directly instead of color names?

Reply:
This is a placeholder for now.